internal/driver/glfw/window.go

```diff
@@ -832,7 +832,7 @@ func (w *window) processFocused(focus bool) {
 func (w *window) triggersShortcut(localizedKeyName fyne.KeyName, key fyne.KeyName, modifier fyne.KeyModifier) bool {
 	var shortcut fyne.Shortcut
 	ctrlMod := fyne.KeyModifierControl
-	if runtime.GOOS == "darwin" {
+	if runtime.GOOS == "darwin" || isMacOSBrowser() {
```

Comment:
This could be simplified - put the GOOS check in place if "false" in the shortcuts_other and call it "isMacOSRuntime instead.
It will also fix the issue where you are running the server on macOS but accessing from a windows browser.

Reply:
renamed to isMacOSRuntime. removed web/browser refererence. put runtime check in the shortcuts_other.